src/devices/cpu/i386/i386segs.hxx

```diff
@@ -2133,7 +2133,7 @@ void i386_device::i386_protected_mode_iret(int operand32)
 			newESP = READ32(ea+12);
 			newSS = READ32(ea+16) & 0xffff;
 			/* Return to v86 mode */
-			//logerror("IRET (%08x): Returning to Virtual 8086 mode.\n",m_pc);
+			//LOGMASKED(LOG_PM_EVENTS, "IRET (%08x): Returning to Virtual 8086 mode.\n",m_pc);
```

Comment:
Should this be uncommented now that it isn’t enabled by default?

Reply:
Yeah, uncommented.